app/build.gradle.kts

```diff
@@ -156,6 +158,9 @@ dependencies {
     implementation(Libs.materialComponents)
     implementation(Libs.flexbox)
 
+    implementation(Libs.playCore)
+    implementation(Libs.playKtx)
```

Comment:
It's often possible to only include the `ktx`-artifact, and get both artifacts for the price of one. Try it out! 🙂